### DISCUSSION TITLE
Host being lost after redirect

### DISCUSSION THREAD

**Discussion by d-shannon** [Kyon147/laravel-shopify#311] [Q&A] [answered]
Hello all,

After redirect I'm getting an appbridge error.

![AppBridgeError](https://github.com/Kyon147/laravel-shopify/assets/500704/730acf3d-5c1a-4324-bfd4-35d1805776d2)

Is anyone else facing this issue?

Thanks!

**Comment by d-shannon** [comment, marked as answer]
Solved my own problem. I was performing a redirect without the host!

**Comment by dandyleon19** [comment]
Hi @d-shannon, could you tell me what the redirect url was or how you did it please? I used the host before, but I still can't find how to get the host

**Reply by d-shannon** [reply]
https://github.com/Kyon147/laravel-shopify/wiki/Authentication-Process#links-routing-and-forms

`return Redirect::tokenRedirect('orders.view', ['id' => 1]);`
